universal/router/React/index.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import React from 'react'
+
+import LayoutSidebar from 'layouts/LayoutSidebar'
+import Markdown from 'components/Markdown'
+import Counter from 'modules/Counter'
+
+import type { Route } from '../types'
+import match from '../matches'
```

Comment:
components is a directory...and i prefer root directories with nothing that can be rendered in and of themselves to be lowercase. All directories within components are uppercase to signify they are components that can be imported and used.

It does need a better API as I would prefer to not have to cherry pick components out of the directory...that's an enhancement on my radar